project/profiles/capacity4more/modules/c4m/content/c4m_content_group/theme/c4m-content-group-header-title.tpl.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * @file
+ * Template for presenting additional information fields.
+ *
+ * Type, Status and Access at content group header title.
+ */
+?>
+<div class="row" id="content-group-header-title">
+  <div class="col-sm-12 col-md-8">
+    <h1><?php print $title;?></h1>
```

Comment:
Problem is that this entire TPL is printed between the H1-tag (so all the divs as well)